android-sdk/build.gradle

```diff
@@ -11,12 +11,17 @@ android {
         // versionCode and versionName are used by the SDK to identify the app to Bridge server
         versionCode 1
         versionName "version 1.0.2, build 1"
+
     }
 
     buildTypes {
+        debug {
+            minifyEnabled false
+            consumerProguardFiles 'proguard-rules.pro'
+        }
         release {
             minifyEnabled false
```

Comment:
Should minifyEnabled be true for release?

Reply:
Disabling minify on the library keeps it intact for dependencies, keeping the lines intact and allowing the app to control minification. At library build time, the library can't tell which methods the app might use. 

If the app has minify enabled, this library will also be minified. The consumerProguardFiles should help with that